t/balancer.t

```diff
@@ -334,8 +334,8 @@ last peer failure: failed 500
 --- grep_error_log eval: qr{last peer failure: \S+ \S+}
 --- grep_error_log_out
 last peer failure: nil nil
-last peer failure: failed 502
-last peer failure: failed 502
+last peer failure: failed 503
+last peer failure: failed 503
```

Comment:
Maybe we should make this an regex so that it also passes with older versions of the nginx core?

Reply:
Sure, I made a change.